client/components/ProductOverview/ProductOverview.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import mockProductStyles from '../../../MockData/ProductStyles';
+import ImageGallery from './ImageGallery';
+import '../../css/ProductOverviewCSS.css';
```

Comment:
I would separate the css import by one line just as a matter of style.

Reply:
refactored.